points/activities/data_selecta.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+
+#
+# This program requires to launch data from https://archive-beta.ics.uci.edu/ml/datasets/daily+and+sports+activities
+# Unzip the data and copy data_selecta.py in a same directory data folder
+# Launch 'python data_selecta.py'
+#
+
+# Will be auto-saved as a xz compressed pickle file
+result_file = 'activities.xz'
```

Comment:
Did you compare the size with `activities.zst`?

Reply:
My _"benchmark"_:
`to_pickle('activities.xz')` : 111 Mo
`to_pickle('activities.bz2')` : 140 Mo
`to_pickle('activities.zst')` : 193 Mo
`to_pickle('activities.zip')` : 199 Mo
`to_pickle('activities.gz')` : 199 Mo
`to_pickle('activities.pkl')` : 410 Mo

`to_csv('activities.csv.xz')` : 144 Mo
`to_csv('activities.csv.gz')` : 160 Mo
`to_csv('activities.csv.zst')` : 170 Mo
`to_csv('activities.csv')` : 418 Mo

Reply:
Pandas pickle file type seems to be a bad idea: https://docs.python.org/3/library/pickle.html
`to_csv('activities.csv.xz')` would be a better candidate, but not sure anymore that github LFS is a good place to store such data. Let me have a look to figshare (sklearn seems to us e this place), zenodo or even recherche.data.gouv.fr

Reply:
Let's go with `to_csv('activities.csv.xz')`. The link to upload the git LFS file is a bit *hackish* and may not be robust, but let's try this solution. @mglisse do you agree ?